### ISSUE TITLE
Updates, (nocoinlist and more.)

### ISSUE THREAD

**Issue by ZeroDot1** [xd4rker/MinerBlock#12]
Take a look at these lists, maybe you can improve the effectiveness of the add-on. 
https://github.com/ZeroDot1/CoinBlockerLists/blob/master/list_browser.txt
https://github.com/keraf/NoCoin/blob/master/src/blacklist.txt
https://github.com/hoshsadiq/adblock-nocoin-list/blob/master/nocoin.txt

**Comment by xd4rker** [comment]
Yes I was thinking of integrating your list to Minerblock. Will work on it soon 😉

Also feel free to add whatever is missing to your list.
https://github.com/xd4rker/MinerBlock/blob/master/assets/filters.txt

Thanks!
